src/components/rooms/FAQ/FAQRoom.tsx

```diff
@@ -12,17 +12,17 @@ const questions = [
     {
       question: "How does registration work?",
       answer:
-        "Since this is an online event, we want to open registration to anyone that is interested. If you're curious, email us for more info at info@vandyhacks.org."
+        "Sign in, and submit your application at apply.vandyhacks.org! You'll also be able to check the status of your application on our registration portal."
```

Comment:
Is it possible to have hyperlinks here? I feel it would be more intuitive to have apply.vh be clickable directly

Reply:
Agreed with this. Since they are directly being put inside a `<div>`, I believe you can format the link as `<a href="apply.vandyhacks.org/">apply.vandyhacks.org</a>` and it should work

Reply:
`<a href="https://apply.vandyhacks.org/" target="_blank" rel="noopener">apply.vandyhacks.org</a>` 
in that case so it doesn't just open in same tab

Reply:
also might as well full send it and hyperlink mailto: for emails?

Reply:
I am unsure as to what you mean by `full send it`. About the `mailto:`, I believe the footer which has links to our social media also has link to email us.

Reply:
I meant while we're at it, what about hyperlinking all info@vandyhacks.org in text to the email?

Reply:
ill sauce in the hyperlinks